bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/channel_types.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solax"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="frequency">
+		<item-type>Number:Frequency</item-type>
+		<label>Electric Frequency</label>
+		<description>Electric frequency of the input/output voltage of the inverter</description>
+	</channel-type>
+	<channel-type id="battery-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Battery Temperature</label>
+		<description>Battery Temperature</description>
```

Comment:
Consider adding icon, semantic tags and state pattern:
```suggestion
		<description>Battery Temperature</description>
		<category>Temperature</category>
		<tags>
			<tag>Measurement</tag>
			<tag>Temperature</tag>
		</tags>
		<state pattern="%d %unit%" readOnly="true"/>
```

Reply:
is the tag freestyle? e.g. I just add random words that describe the channel?

Reply:
How do you add an icon? I don't see it in the schema...

Reply:
The `category` tag adds it.